lib/Rx/Operator/MapOperator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+
+namespace Rx\Operator;
+
+
+use Rx\ObservableInterface;
+use Rx\Observer\CallbackObserver;
+use Rx\ObserverInterface;
+use Rx\SchedulerInterface;
+
+class MapOperator implements OperatorInterface
+{
+
+    /** @var callable  */
+    private $selector;
+
+    /**
+     * MapOperator constructor.
+     * @param $selector
+     */
+    public function __construct($selector)
+    {
+        if ( ! is_callable($selector)) {
```

Comment:
Nvm. Let's keep it here and fix it in one sweep. Filed an issue for it: https://github.com/asm89/Rx.PHP/issues/14.